help/de/Help_ch_GUIMenus.xml

```diff
@@ -6000,12 +6000,14 @@ url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.
 
         <table frame="topbot" id="budget-menus-actions-items">
           <title>Budget - Aktionen-Menü - Einträge im Standardmenü die sich auf Budgets beziehen.</title>
-
-          <tgroup align="left" cols="2">
+<!-- FIXME source: number of columns -->
+          <tgroup align="left" cols="3">
```

Comment:
Do you really need a 3. column?
If the width is not adjusted as you like, you can hardcode itt by `width="25%"`etc.

Reply:
… and remove the corresponding `namest`, `nameend`attributes from the cells.

Reply:
you are right, I can also place the note from the third column in front of the table.